core/transport/src/main/proto/grpc/job_master.proto

```diff
@@ -305,6 +325,8 @@ message RegisterJobWorkerPOptions {}
 message RegisterJobWorkerPRequest {
   optional grpc.WorkerNetAddress workerNetAddress = 1;
   optional RegisterJobWorkerPOptions options = 2;
+  optional string version = 3;
```

Comment:
Would you like to add worker process start time to show the which worker has been restarted or new coming.